src/utils/airtable.js

```diff
@@ -16,13 +17,14 @@ function transformCategory(data) {
     data: { Name, Count, Cover, Parent },
   } = data
   const parent = transformCategory(Parent?.[0])
+  const cover = Cover?.localFiles?.[0]?.childImageSharp
 
   return {
     id,
     name: Name,
     fullName: parent ? [parent.name, Name].join(" > ") : Name,
     count: Count,
-    cover: Cover?.localFiles?.[0]?.childImageSharp?.fluid,
+    cover: cover?.fluid || cover?.resize,
```

Comment:
we might want to move `i.resize || i.fixed || i.fluid)` into a small helper so it is shared

Reply:
@jasonLaster yeah this smells to me. I almost think we should not traverse that deep when transforming these, so it's explicit what kind of image we're dealing with from the caller.